addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -175,10 +178,14 @@ export class ControlPanel extends Component {
         onMounted(async () => {
             if (this.state.embeddedInfos.embeddedActions?.length > 0) {
                 // If there is no visible embedded actions, the current action (if it exists) is put by default
+                const embeddedActionKey =
+                    this.state.embeddedInfos.currentEmbeddedAction?.id || false;
                 if (
-                    !Object.keys(this.state.embeddedInfos.visibleEmbeddedActions).includes("false")
+                    !Object.keys(this.state.embeddedInfos.visibleEmbeddedActions).includes(
+                        embeddedActionKey.toString()
```

Comment:
Is this `toString()` really necessary ?

Reply:
Yes, because Object.keys stores the keys as strings